examples/shapes/shapes_math_sine_cosine.c

```diff
@@ -0,0 +1,180 @@
+/*******************************************************************************************
+ *
+ *   raylib [shapes] example - math sine cosine
+ *
+ *   Port of the p5.js "sine and cosine" demo to raylib + raygui
+ *   Sources: https://p5js.org/examples/angles-and-motion-sine-cosine/
+ *
+ *   Demonstrates uniform circular motion and the correspondence between
+ *   the unit circle and sine / cosine graphs. Right panel contains a
+ *   play/pause toggle implemented with raygui.
+ * 
+ *   Example originally created with raylib 5.6-dev, last time updated with raylib 5.6-dev
+ *
+ *   Copyright (c) 2025 Olivier (@oqu)
+ *
+ ********************************************************************************************/
+
+#include "raylib.h"
+
+// Required for GUI controls
+#define RAYGUI_IMPLEMENTATION
+#include "raygui.h"
+
+#include <math.h>
+
+int main(void)
+{
+    // Window initialization
```

Comment:
Note that Init, Update, Draw sections should be defined by:
```//--------------------------------------------------------------------------------------```
following the template structure: https://github.com/raysan5/raylib/blob/master/examples/examples_template.c